README.md

```diff
@@ -1,208 +1,262 @@
+Choose Push Notification service certificate. As you may see, there are two types of certificates: _development_ and _production_. Note that the development type certificate can only work with the sandbox environment. More about this explained later on in this guide.
 
-<img src="assets/images/step2.png"/>
+<img src="assets/images/apns/Cert-1.png"/>
 
-### Step 3 - Select Devices
-Select the devices that you want to deploy your app to and upload splash screen images for the devices you have selected. 
+### Step 3 - Select App ID
 
-Press the button publish to appstore and you will be asked whether you want to build in the cloud or do it yourself. Choose do it yourself and press the Download Phonegap Build Package.
+Pick your App ID from the dropdown list. If your app is not on the list, then you need to check your App ID entity. Most likely the push notification service is not turned on for the app.
 
+<img src="assets/images/apns/Cert-2.png"/>
 
-<img src="assets/images/step3.png"/>
+### Step 4 - CSR file
 
-### Step 4 - Download phonegap.zip
+On the next step you should be asked for your CSR file (Certificate Signing Request). You may use the same CSR that you used to create app signing certificate. If you don't have one, please follow the instruction within this step description.
 
-Once the phonegap.zip is downloaded, unzip it into a folder and open up the config.xml. You will need to edit the config.xml so that you can include additional phonegap plugins. The plugin we will need to include is the phonegap [push plugin](https://github.com/phonegap-build/PushPlugin).
+<img src="assets/images/apns/Cert-3.png"/>
 
-The code you will need to include is:
-`<gap:plugin name="com.phonegap.plugins.pushplugin" version="2.4.0" />`
+### Step 5 - Download the certificate
 
-<img src="assets/images/step4.png"/>
+Download your Apple Push Notification service SSL certificate and add it to your Keychain.
 
-### Step 5 - Configure xml
-Once you have edited the config.xml you should have everything necessary for your application to work. You will now need to zip up your files and upload the zipped file to [phonegap build](https://build.phonegap.com).
+This certificate needs to be converted into `.p12` format. If you don’t know how to do it, please refer to [this page]  (https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/MaintainingCertificates/MaintainingCertificates.html).
 
-<img src="assets/images/step5.png"/>
+### Step 6 - Configure APNs on your application
+
+For the last step you need to configure the APNs setting within your application. This can be done by logging into your application as a user with Administrator role and opening the Apple Administration page that was set up in step 7 of the Implementation Guide.
```

Comment:
configure the setting seems rather double... would just replace with 'you need to configure APNs within your application'

Might be useful to link to the step through text link? Minor detail though.